src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Bed.java

```diff
@@ -0,0 +1,40 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * Class bed has 1 private member and one inherited from the abstract class AbstractCommodity
+ * methods - getters,setters,constructor and implemented method prepare()
+ */
+
+
+public class Bed extends AbstractCommodity {
+
+    private BedSize numberOfPersonas;
```

Comment:
It's good that we changed the type, but let's change the field name as well, so it matches. Maybe rename numberOfPersonas to bedType?